### DISCUSSION TITLE
I2C Display refresh in real time

### DISCUSSION THREAD

**Discussion by grappini** [arendst/Tasmota#12823] [Q&A] [answered]
For the moment the I2C display data can be refreshed only by teleperiod with a minimum of 10 second?
Is there any other way to refresh it faster? Like in web interface?
Could the .ino file of the device sensor be modified so that when it send the data to the web interface to do it also to the i2c display?

**Comment by barbudor** [comment, marked as answer]
Instead of using an automatic mode, you can make it manually through rules

**Reply by grappini** [reply]
Can you post an exemple?

**Reply by barbudor** [reply]
like in the docs https://tasmota.github.io/docs/Displays/#rule-examples-for-scripting-examples-see-scripting-docs
but without the `tele-` prefix

**Reply by grappini** [reply]
Like this?
Rule 1
on BMP280#Pressure do DisplayText [f1p10x140y5]%value% hPa endon

And is refreshing the i2c display every time the bmp sensor values change?

**Reply by barbudor** [reply]
It will refresh the display the same rate as the Web UI which is the max rate in the driver

**Reply by grappini** [reply]
Thanks. I will provide feedback after receiving and testing my display.
